src/config/options.rs

```diff
@@ -523,6 +528,7 @@ impl From<StyleEdition> for rustc_span::edition::Edition {
             StyleEdition::Edition2018 => Self::Edition2018,
             StyleEdition::Edition2021 => Self::Edition2021,
             StyleEdition::Edition2024 => Self::Edition2024,
+            StyleEdition::Edition2027 => Self::Edition2024,
```

Comment:
Let's add a TODO comment here explaining that `rustc_span::edition::Edition` doesn't have an `Edition2027` variant.